Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth
# Nazjatar artifacts

## Changelog:
- Added naga artifacts.


# How to test:
Use console command "add_artifact artifact name" to add all artifacts and make sure they have icon and localization.
**List of artifact names:**
- `naga_scimitar`
- `naga_mace`
- `naga_scepter`
- `naga_trident`
- `statue_of_azshara `
- `queens_decree`
- `eternal_bouquet`
- `taxidermied_murloc`

You can check balance here: https://docs.google.com/spreadsheets/d/1xDjRHfG-ovKVcEQVfv02BsK3j0dgwS2rfH7KJ4GirA0/edit#gid=1038522838
